math-scala/src/main/scala/org/apache/mahout/math/drm/package.scala

```diff
@@ -148,6 +148,42 @@ package object drm {
   def drmSampleKRows[K](drmX: DrmLike[K], numSamples: Int, replacement: Boolean = false): Matrix =
     drmX.context.engine.drmSampleKRows(drmX, numSamples, replacement)
 
+  /**
+    * Convert a sampled DRM into a Tab Separated Vector (TSV) to be loaded into an R-DataFrame
+    * for plotting and sketching
+    * @param drmX - DRM
+    * @param samplePercent - Percentage of Sample elements from the DRM to be fished out for plotting
+    * @tparam K
+    * @return TSV String
+    */
+  def sampleMatrixToTSV[K](drmX: DrmLike[K], samplePercent: Double = 1): String = {
+
```

Comment:
Minor point: maybe rename to `drmSampleToTSV` or something along those lines so that it is obvious that its a DRM and not a Matrix?  Other than that +1 from me.